### PR TITLE
Replace the existing stale bot with the new one

### LINKED ISSUE #15403: Replace the existing stale bot with the new one
## Provide a description of the task

Replace the existing stale bot with the new one.

I think, we can create a new task on CI that will be executed daily.

_Part of ckeditor/ckeditor5#15399._

### PR DESCRIPTION
**Pull request by psmyrek**
### Suggested merge commit message ([convention](https://ckeditor.com/docs/ckeditor5/latest/framework/contributing/git-commit-message-convention.html))

Internal: Removed the existing stale bot and added the CI workflow with the configuration file for the new one. Closes #15403.

---

### Additional information

_For example – encountered issues, assumptions you had to make, other affected tickets, etc._

### CODE REVIEW

File: .stale-bot.js
Comment (pomek) [inline .stale-bot.js:1]: I wonder where to keep this file to add it to the root directory that contains many such files.

My first thought was the `.github/` directory, but the configuration has nothing to do here. Perhaps the `.circle/` folder fits better, WDYT?

Also, do you know if the file passed eslint? As a pure JS code, strings should be wrapped in `'`, and the object's keys do not need it.

Comment (psmyrek) [reply .stale-bot.js:1]: Moved to `.circleci/.stale-bot.js` and aligned to ESLint rules.

**Comment by psmyrek** [comment]
CI is 🔴 because this PR requires changes (new `ckeditor5-dev-stale-bot` package) from https://github.com/ckeditor/ckeditor5-dev/pull/933.

**Comment by pomek** [comment]
Now, CI should not complain – https://github.com/ckeditor/ckeditor5-dev/releases/tag/v39.4.0.

Let's see.

**Review by pomek — APPROVED** [review]
LGTM.

**Pull request merged**